app/views/layouts/landing_page_layout.html.erb

```diff
@@ -167,10 +167,10 @@
               <a class="govuk-link govuk-link--no-visited-state app-mobile-nav__link" href="/main/Round1Version1/maths/counting">Counting</a>
             </li>
             <li class=" app-mobile-nav__subnav-item">
-              <a class="govuk-link govuk-link--no-visited-state app-mobile-nav__link" href="#">Number patterns</a>
+              <a class="govuk-link govuk-link--no-visited-state app-mobile-nav__link" href="#">Numbers in context</a>
             </li>
             <li class=" app-mobile-nav__subnav-item">
-              <a class="govuk-link govuk-link--no-visited-state app-mobile-nav__link" href="#">Spatial reasoning</a>
+              <a class="govuk-link govuk-link--no-visited-state app-mobile-nav__link" href="#">Reasoning</a>
```

Comment:
This whole menu needs to be deleted

It is hidden because of the CSS, but the html is here as a hang over from when it was a place holder for the mobile menu

I think it should be deleted in this PR

Reply:
gone